visualizations/rhods-notebooks/horreum_lts_store.py

```diff
@@ -162,26 +162,29 @@ def build_lts_payloads() -> dict:
         start_time: datetime.datetime = results.start_time
         end_time: datetime.datetime = results.end_time
 
-        output = {
-            "$schema": "urn:rhods-matbench-upload:3.0.0",
+        payload = {
+            "$schema": "urn:rhods-notebooks:1.0.0",
             "data": {
                 "users": _decode_users(results),
                 'rhods_version': results.rhods_info.version,
                 'ocp_version': results.sutest_ocp_version,
                 'metrics': _gather_prom_metrics(entry),
                 'thresholds': results.thresholds,
-                'config': results.test_config.yaml_file
+                'config': results.test_config.yaml_file,
+                "cluster_info": _parse_entry(entry.results.rhods_cluster_info),
             },
             "metadata": {
-                "test": "rhods-notebooks",
+                "presets": results.test_config.get("ci_presets.names") or ["no_preset_defined"],
+                "test": results.test_config.get('tests.identifier', "unknown"),
```

Comment:
```suggestion
                "test": results.test_config.get('tests.identifier', "unspecified"),
```

Reply:
🤔 you didn't apply this suggestion 🤔 
and to work, you need to change the key:
```suggestion
                "test": results.test_config.get('tests.notebooks.identifier', "missing"),
```
can you also set, in the config file:
```
tests:
  notebooks:
    identifier: unspecified
```
so that there should always be a value